database/sqlite/create-schema.migrations.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE drs_object (
+    id INTEGER PRIMARY KEY,
+    description TEXT,
+    created_time DATETIME,
+    mime_type TEXT,
+    name TEXT,
+    size UNSIGNED BIG INT,
+    updated_time DATETIME,
+    version TEXT
+);
+
+CREATE TABLE drs_object_access_method (
+    drs_object_id INTEGER,
+    access_id TEXT,
+    access_url TEXT,
+    region TEXT,
+    type TEXT CHECK (type IN ('s3', 'https', 'file')),
+    FOREIGN KEY(drs_object_id) REFERENCES drs_object(id)
+);
+
+CREATE TABLE drs_object_aliases (
+    drs_object_id INTEGER,
+    alias TEXT,
+    PRIMARY KEY (drs_object_id, alias),
+    FOREIGN KEY(drs_object_id) REFERENCES drs_object(id)
+);
+
+CREATE TABLE drs_object_checksum (
+    drs_object_id INTEGER,
+    checksum TEXT,
+    type TEXT CHECK (type IN ('md5', 'etag', 'crc32c', 'trunc512', 'sha1', 'sha-256')),
+    PRIMARY KEY (drs_object_id, type),
+    FOREIGN KEY(drs_object_id) REFERENCES drs_object(id)
+);
+
+CREATE TABLE drs_object_bundle (
+    parent_id INTEGER,
+    child_id INTEGER,
+    PRIMARY KEY (parent_id, child_id),
+    FOREIGN KEY(parent_id) REFERENCES drs_object(id),
+    FOREIGN KEY(child_id) REFERENCES drs_object(id)
+);
```

Comment:
let's put some commands in `populate-tables.migrations.sql` to add some phenopacket files in the test directory to the db. please incorporate 1-2 bundles as well here. I want to be able to run SQL queries from inside the container as a success metric